src/handlers/interventions-processor-handler.ts

```diff
@@ -183,6 +183,13 @@ function getInterventionName(recordBody: TxMAIngressEvent): EventsEnum {
   return recordBody.event_name as EventsEnum;
 }
 
+/**
+ * Function to ascertain that the time stamp is after the intervention has been applied.
```

Comment:
Could we specify here:
Function to ascertain that the event timestamp is after the latest intervention applied on the account